alephium/contracts/token_bridge/token_bridge.ral

```diff
@@ -22,6 +22,7 @@ Contract TokenBridge(
     refundAddress: Address
 ) extends Constants(), TokenBridgeConstants() {
     const TokenBridgeModule = 0x546f6b656e427269646765 // module identifier: "TokenBridge"
+    const MinimalConsistencyLevel = 10
```

Comment:
It's fixed in the ETH contract, if we allow the user to set the `consistencyLevel`, we need to make sure `consistencyLevel > MinmalConsistencyLevel` to avoid problems caused by reorg.

Reply:
Maybe 10 is a bit too much, users need to wait about 10 minutes